hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/container/balancer/TestContainerBalancer.java

```diff
@@ -142,9 +143,10 @@ public void testStartBalancerStop() throws Exception {
     stopBalancer();
     assertSame(ContainerBalancerTask.Status.STOPPED, containerBalancer.getBalancerStatus());
 
-    assertThrows(Exception.class,
-        () -> containerBalancer.stopBalancer(),
-        "Exception should be thrown when stop again");
+    // If the balancer is already stopped, the stop command should do nothing
+    // and return successfully with exit code 0 as stopBalancer is idempotent
+    assertDoesNotThrow(() -> containerBalancer.stopBalancer());
```

Comment:
Let's move one of these stop calls to the beginning of this test and add a comment mentioning that stop should not throw because it's idempotent.